<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FacilityInfo</class>
 <widget class="QDialog" name="FacilityInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
    <pointsize>18</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Facility Info</string>
  </property>
  <widget class="QLabel" name="bg_dark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #22223b;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg_white">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f2e9e4;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>-10</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;</string>
   </property>
   <property name="text">
    <string>Facility Info 1F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>630</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a01_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>A01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a02_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>A02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a03_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>A03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a04_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>A04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a05_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>A05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a06_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>A06</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b01_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>B01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b02_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>B02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b03_button">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>160</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>B03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b04_button">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>370</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>B04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to2fButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>630</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15;</string>
   </property>
   <property name="text">
    <string>Go to 2F</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
